src/Components/ProjectDetails/ProjectDetails.jsx

```diff
@@ -12,14 +12,44 @@ import Edit from "../../assets/Edit.png";
 import Link from "../../assets/Link.png";
 import Navigate from "../../assets/Navigate.png";
 import ProjectModal from "../ProjectModal/ProjectModal";
+import { Modal, Button } from "react-bootstrap";
+
+
+const DeleteConfirmation = ({ showModal, hideModal, confirmModal, id}) => {
```

Comment:
Hey, just for React convention's sake, can we have this component moved to a separate file in the /components folder and then get it imported here.🤞
Also, please try to vertically center the modal and see if that somehow is a bit better..especially considering the fact that the delete icon is at the bottom right.
Ref: https://getbootstrap.com/docs/4.1/components/modal/#vertically-centered

Reply:
done